org.apache.spark:spark-streaming-kafka_2.11:jar:2.0.0 missing dependency

The maven build does not work due to a missing dependency as of 1dc2331
see also #351

```
[ERROR] Failed to execute goal on project sparkbench-streaming: Could not resolve dependencies for project com.intel.hibench.sparkbench:sparkbench-streaming:jar:6.0-SNAPSHOT: Failed to collect dependencies at org.apache.spark:spark-streaming-kafka_2.11:jar:2.0.0: Failed to read artifact descriptor for org.apache.spark:spark-streaming-kafka_2.11:jar:2.0.0: Could not transfer artifact org.apache.spark:spark-streaming-kafka_2.11:pom:2.0.0 from/to scala-tools.org (https://oss.sonatype.org/content/groups/scala-tools/): Access denied to: https://oss.sonatype.org/content/groups/scala-tools/org/apache/spark/spark-streaming-kafka_2.11/2.0.0/spark-streaming-kafka_2.11-2.0.0.pom , ReasonPhrase:Forbidden. -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :sparkbench-streaming
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -25,7 +25,7 @@ cache:
   directories:
   - $HOME/.m2
 script:
- # - mvn clean package -q -Dmaven.javadoc.skip=true -Dspark=2.0
+  - mvn clean package -q -Dmaven.javadoc.skip=true -Dspark=2.0
   - mvn clean package -q -Dmaven.javadoc.skip=true
   - sudo -E ./travis/configssh.sh
   - sudo -E ./travis/restart-hadoop-spark.sh

--- a/.travis.yml
+++ b/.travis.yml
@@ -3,7 +3,6 @@ jdk:
   - openjdk7
   - oraclejdk8
 install: 
-  - cd src
 before_script:
   - "echo $JAVA_OPTS"
   - "export JAVA_OPTS=-Xmx512m"

--- a/.travis.yml
+++ b/.travis.yml
@@ -2,7 +2,7 @@ language: java
 jdk:
   - openjdk7
   - oraclejdk8
-install: 
+install:
 before_script:
   - "echo $JAVA_OPTS"
   - "export JAVA_OPTS=-Xmx512m"

--- a/.travis.yml
+++ b/.travis.yml
@@ -8,7 +8,6 @@ before_script:
   - "echo $JAVA_OPTS"
   - "export JAVA_OPTS=-Xmx512m"
 env:
-  - SPARK_VERSION=1.5
   - SPARK_VERSION=1.6
 script:
-  - mvn clean package -q -Dmaven.javadoc.skip=true -Dspark${SPARK_VERSION} -DMR2
+  - mvn clean package -q -Dmaven.javadoc.skip=true

--- a/pom.xml
+++ b/pom.xml
@@ -22,8 +22,6 @@
     <mahout.version>0.9</mahout.version>
     <uncommons-maths.version>1.2.2a</uncommons-maths.version>
     <junit.version>3.8.1</junit.version>
-    <spark.version>1.6.0</spark.version>
-    <spark.bin.version>1.6</spark.bin.version>
     <hadoop.mr2.version>2.4.0</hadoop.mr2.version>
     <hadoop.mr1.version>1.2.1</hadoop.mr1.version>
     <scala-maven-plugin.version>3.2.0</scala-maven-plugin.version>

--- a/sparkbench/pom.xml
+++ b/sparkbench/pom.xml
@@ -116,6 +116,19 @@
     </profile>
 
     <profile>
+      <id>defaultSpark</id>
+      <properties>
+        <spark.version>1.6.0</spark.version>
+        <spark.bin.version>1.6</spark.bin.version>
+      </properties>
+      <activation>
+        <property>
+          <name>!spark</name>
+        </property>
+      </activation>
+    </profile>
+
+    <profile>
       <id>spark1.6</id>
       <properties>
         <spark.version>1.6.0</spark.version>

--- a/sparkbench/streaming/pom.xml
+++ b/sparkbench/streaming/pom.xml
@@ -36,6 +36,22 @@
 
   <profiles>
     <profile>
+      <id>defaultSpark</id>
+      <dependencies>
+        <dependency>
+          <groupId>org.apache.spark</groupId>
+          <artifactId>spark-streaming-kafka_${scala.binary.version}</artifactId>
+          <version>1.6.0</version>
+        </dependency>
+      </dependencies>
+      <activation>
+        <property>
+          <name>!spark</name>
+        </property>
+      </activation>
+    </profile>
+
+    <profile>
       <id>spark1.6</id>
       <dependencies>
         <dependency>

--- a/sparkbench/streaming/pom.xml
+++ b/sparkbench/streaming/pom.xml
@@ -32,10 +32,41 @@
       <version>${spark.version}</version>
       <scope>provided</scope>
     </dependency>
-    <dependency>
-      <groupId>org.apache.spark</groupId>
-      <artifactId>spark-streaming-kafka_${scala.binary.version}</artifactId>
-      <version>${spark.version}</version>
-    </dependency>
   </dependencies>
+
+  <profiles>
+    <profile>
+      <id>spark1.6</id>
+      <dependencies>
+        <dependency>
+          <groupId>org.apache.spark</groupId>
+          <artifactId>spark-streaming-kafka_${scala.binary.version}</artifactId>
+          <version>1.6.0</version>
+        </dependency>
+      </dependencies>
+      <activation>
+        <property>
+          <name>spark</name>
+          <value>1.6</value>
+        </property>
+      </activation>
+    </profile>
+
+    <profile>
+      <id>spark2.0</id>
+      <dependencies>
+        <dependency>
+          <groupId>org.apache.spark</groupId>
+          <artifactId>spark-streaming-kafka-0-8_2.11</artifactId>
+          <version>2.0.0</version>
+        </dependency>
+      </dependencies>
+      <activation>
+        <property>
+          <name>spark</name>
+          <value>2.0</value>
+        </property>
+      </activation>
+    </profile>
+  </profiles>
 </project>

